dev/bots/test/analyze-test-input/root/packages/flutter/lib/bar.dart

```diff
@@ -15,6 +15,13 @@ class Foo {
 
   @_debugOnly
   final Map<String, String>? bar = kDebugMode ? null : <String, String>{};
+
+  // dart format off
```

Comment:
Maybe leave a comment here explaining why format is turned off (e.g. to check that the annotation also work properly if the annotated things goes over multiple lines)?